### DISCUSSION TITLE
Markdown tables to csv(s) ?

### DISCUSSION THREAD

**Discussion by JulienParis** [hng/tech-coops#147] [Ideas]
Just a proposal here =>

**Who would be interested in "migrating" the lists from the `README.md` to one or several `csv` file(s) ?** 

As I'm writing this proposal I'd see the following _pros_ and _cons_ for this idea : 

- **_pros_ for a `csv` file** :
  - better format for reuses ;
  - easier to share as a file ;
  - better format if the lists begins to contain more than dozen lines. A `md` file containing hundred-lines-long table a might be not very user-friendly.
- **_cons_ against a `csv` file** (keep the lists on the `REAME`) :
  - the `README` is directly shown on the repo ;
  - perhaps trickier to constribute to a `csv` (no commas in cells)
- **neither _pro_ nor _con_ / maintenance** : 
  - to avoid confusion and to avoid entering twice the values, if one or several `csv` file compile all coops the `README` should not include the lists... so the `README` would be more like a summary than containing the data itself... The risk here - if done without coordination - is that also would have some impacts on @hng 's [website](http://tech-coops.xyz/), and that would be a shame given the amount of effort he's put into this !

**Note** :

I'm proposing the `csv` approach because in France there's a quite structured initiative called [CoopTech](https://cooptech.fr/), and we're about to make a relatively large registry of (tech) coops. This census might contains more than a hundred (hundreds perhaps) entries. 

What do you think about this idea ?

**Comment by hng** [comment]
Thanks for the write up! I'd have to thing this one a bit. Creating the website was already one step to having a bit more possibilities than with the GitHub markdown view.

Indeed an addition of more than 100 entries from just one country would make the list as it is now less useful (but that's also the case if it grows in general).

I think the main question would be if and how well we still want/need to support people that read the list through GitHub.

**Comment by carlos-reynosa** [comment]
I had suggested this in another discussion:

https://github.com/hng/tech-coops/discussions/80#discussioncomment-379724
